meetings/2023/2023-08-08.md

```diff
@@ -0,0 +1,101 @@
+# OpenJS Foundation Cross Project Council Meeting 2023-08-08
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=BZnz21hQ83E
+* **GitHub Issue**: https://github.com/openjs-foundation/cross-project-council/issues/1132
+
+## Present
+
+* Joe Sepi (@joesepi)
+* Ben (@bensternthal)
+* Rick Markins (@rxmarbles)
+* Claudio Wunder (@ovflowd)
+* Sean Larkin (@thelarkinn)
+* Rich Trott (@Trott)
+* Tobie Langel (@tobie)
+* Christian Bromann (@christian-bromann)
+* Adrian Estrada (@edsadr)
+* Robin Ginn (@rginn)
+* Tierney Cyren (@bnb)
+* Sara Chipps (@SaraJo)
+* Kylie Wagar-Dirks (@kyliewd) 
+
+## Agenda
+
+### Announcements
+
+* Claudio - nodejs collab summit registration is live
+https://github.com/openjs-foundation/summit/issues/360
+
+### Board Meeting Updates
+
+*Extracted from **cross-project-council-agenda** labeled issues and pull requests from the **openjs-foundation org** prior to the meeting.
+
+### openjs-foundation/cross-project-council
+
+* Upcoming CPC Elections [#1126](https://github.com/openjs-foundation/cross-project-council/issues/1126)
+  * Ben work with Joe on elections, ensure nomination respondes are public (Tobie request)
+
+* Code of Conduct Plan of Action [#1122](https://github.com/openjs-foundation/cross-project-council/issues/1122)
+  * progress is happening; take a look at open PRs; any help is appreciated
+
+* add @edsadr as a regular member to CPC [#1121](https://github.com/openjs-foundation/cross-project-council/pull/1121)
+  * Welcome to Adrian; as end user director
```

Comment:
```suggestion
  * Welcome to Adrian; as end-user director
```

Reply:
The actual title is End User Director (without the hyphen), so let's leave it as such.